packages/theming/apple-theme/src/fluentAppleColors.macos.ts

```diff
@@ -0,0 +1,490 @@
+    successShade20: '#0F7A0B',
+    successShade30: '#0B5A08',
+    successTint10: '#27AC22',
+    successTint20: '#5EC65A',
+    successTint30: '#A7E3A5',
+    successTint40: '#CEF0CD',
+    warningPrimary: '#FFD335',
+    warningShade10: '#E6BE30',
+    warningShade20: '#C2A129',
+    warningShade30: '#8F761E',
+    warningTint10: '#FFD94E',
+    warningTint20: '#FFE586',
+    warningTint30: '#FFF2C3',
+    warningTint40: '#FFF8DF',
+
+    brandBackgroundDisabled: '#252525',
+    brandForegroundDisabled: '#252525',
+    neutralBackground2: '#FFFFFF',
+    neutralBackground3: '#000000',
+    neutralForeground2: '#000000',
+    neutralForeground3: '#272727',
+    neutralInverted: '#FFFFFF',
+    neutralStroke2: '#000000',
+  };
+}
+
+export function getStockAppleDarkPalette(): ApplePalette {
+  return {
+    blue10: '#4F6BED',
+    blueMagenta20: '#8764B8',
+    blueMagenta30: '#5C2E91',
+    communicationBlue: '#0086F0',
+    communicationBlueShade10: '#1890F1',
+    communicationBlueShade20: '#3AA0F3',
+    communicationBlueShade30: '#6CB8F6',
+    communicationBlueTint10: '#0078D4',
+    communicationBlueTint20: '#004C87',
+    communicationBlueTint30: '#043862',
+    communicationBlueTint40: '#092C47',
+    cyan20: '#038387',
+    cyan30: '#005B70',
+    cyanBlue10: '#0078D4',
+    cyanBlue20: '#004E8C',
+    dangerPrimary: '#E83A3A',
+    dangerShade10: '#EA4C4C',
+    dangerShade20: '#EE6666',
+    dangerShade30: '#F28C8C',
+    dangerTint10: '#CC3333',
+    dangerTint20: '#8B2323',
+    dangerTint30: '#461111',
+    dangerTint40: '#250909',
+    gray20: '#69797E',
+    gray25: '#F8F8F8',
+    gray30: '#7A7574',
+    gray40: '#393939',
+    gray50: '#F1F1F1',
+    gray100: '#E1E1E1',
+    gray200: '#C8C8C8',
+    gray300: '#ACACAC',
+    gray400: '#919191',
+    gray500: '#6E6E6E',
+    gray600: '#404040',
+    gray700: '#303030',
+    gray800: '#292929',
+    gray900: '#212121',
+    gray950: '#141414',
+    green10: '#498205',
+    green20: '#0B6A0B',
+    magenta10: '#C239B3',
+    magenta20: '#881798',
+    magentaPink10: '#E3008C',
+    orange20: '#CA5010',
+    orange30: '#8E562E',
+    orangeYellow20: '#986F0B',
+    pinkRed10: '#750B1C',
+    presenceAvailable: '#92C353',
+    presenceAway: '#F8D22A',
+    presenceBlocked: '#D74553',
+    presenceBusy: '#D74553',
+    presenceDnd: '#D74553',
+    presenceOffline: '#979593',
+    presenceOof: '#E959D9',
+    presenceUnknown: '#979593',
+    red10: '#D13438',
+    red20: '#A4262C',
+    successPrimary: '#979593',
+    successShade10: '#20BA53',
+    successShade20: '#3BC569',
+    successShade30: '#67D48B',
+    successTint10: '#0D9D3D',
+    successTint20: '#096B29',
+    successTint30: '#043615',
+    successTint40: '#021D0B',
+    warningPrimary: '#FFC328',
+    warningShade10: '#FFC83E',
+    warningShade20: '#FFDD15',
+    warningShade30: '#FFDD87',
+    warningTint10: '#E0AB24',
+    warningTint20: '#997518',
+    warningTint30: '#4D3A0C',
+    warningTint40: '#291F07',
```

Comment:
Can we fetch the colors from FUA instead of hardcoding them?

Reply:
That was my original intention, but I think it ends up being more indirection than it's worth. If we fetch them from FUA, we introduce a cocoapod dependency. We would still need to update the pod to increment the colors or add mappings for new colorsm so it's not really that we get the newest colors for free. Furthermore, this whole palette interface is subject to change as we move to the design token pipeline, and it's easier to rewrite and verify JS code in one repo, than JS and native code in two repos.

Reply:
Edit: Will need to make a native theming module to get access to the [macOS System effects API](https://developer.apple.com/documentation/appkit/nscolor/2998826-withsystemeffect). Will leave as a follow up